### DISCUSSION TITLE
wrong v-for Map typings?

### DISCUSSION THREAD

**Discussion by j4k0xb** [vuejs/language-tools#542] [Q&A] [answered]
when iterating through a map, its typically done like this:
```html
<div v-for="[key, value] in map"
```
My attempt of also accessing the index:
```html
<div v-for="([key, value], a, b) in map"
```
`a` is typed as `keyof Map` (all the 'clear', 'delete', etc. methods)
and `b` is typed as a `number`

but actually `a` is the index and `b` is always undefined (when printing it)
bug in volar?

**Comment by johnsoncodehk** [comment, marked as answer]
This is a bug, please track #544.

Reference: https://github.com/vuejs/vue-next/blob/ca17162e377e0a0bf3fae9d92d0fdcb32084a9fe/packages/runtime-core/src/helpers/renderList.ts#L77-L80
